src/swarm/neo/connection/RequestOnConnBase.d

```diff
@@ -243,6 +244,112 @@ abstract class RequestOnConnBase
 
         /***********************************************************************
 
+            Bitfield enum to specify the events to wait on. Passed to the
+            nextEvent() method.
+
+        ***********************************************************************/
+
+        public enum Flags
```

Comment:
I would name this enum differently, since it's now a member of `EventDispatcher`, but it's really only relevant for `nextEvent`.